content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,164 @@
+---
+title: TiKV 3.0 GA Release & Benchmarks
+date: 2019-07-08
+---
+
+Today we're proud to announce the general availability of TiKV 3.0! Before release, TiKV 3.0 underwent a rigorous testing regime, including the [official Jepsen test](https://www.pingcap.com/blog/tidb-passes-jepsen-test-for-snapshot-isolation-and-single-key-linearizability/) with TiDB.
+
+In this version, we tackled many problems of stability at massive scale. Whether spanning hundreds of nodes or storing over a trillion key-value pairs, we've seen our users put TiKV to the test in serious, real-world, production scenarios. In 3.0, we've applied the lessons learned from these deployments to bring a host of new features that can better support users' growing demands.
```

Comment:
Instead of "problems" this could be spun more positively to say e.g. "this release is stable at greater scales than ever before". Not the precise wording, but more positive than using the word "problems".

Reply:
Like previous comment, the first two paragraphs are the time to deliver the good news. It's ok to admit that previous releases had "problems", but maybe not in the lead.

Reply:
And I would probably re-order these two points. I think the main message is "stable at greater scales than ever before". Part of how we achieved that is by doing rigorous jepsen test. It's probably not that we did a jepsen test that is the main message.

Reply:
This was similar verbatim to https://pingcap.com/blog/tidb-3.0-announcement/ but I generally agree. Let me change it.